geopandas/array.py

```diff
@@ -985,7 +986,26 @@ def _formatter(self, boxed=False):
             ``boxed=True``.
         """
         if boxed:
-            return str
+            import geopandas
+
+            precision = geopandas.options.display_precision
+            if precision is None:
+                # dummy heuristic based on 10 first geometries that should
```

Comment:
Once CRS class #1101 will be adopted, we can just ask pyproj if CRS is projected or not, instead of doing this. Just linking it here for now so we can change that later.

Reply:
Yes, indeed (although we might still want to keep the current heuristic as fallback if the data has no CRS set)